packages/apm/src/integrations/tracing.ts

```diff
@@ -352,6 +519,32 @@ export class Tracing implements Integration {
     }
   }
 
+  /**
+   * Adds offset to the span
+   *
+   * @param measureName name of the performance measure
+   * @param span Span to add data.offset to
+   */
+  private static _addOffsetToSpan(measureName: string, span: SpanClass): void {
+    if (global.performance) {
+      const name = `#sentry-${measureName}`;
+      performance.measure(name);
+      const measure = performance.getEntriesByName(name).pop();
+      if (measure) {
+        span.setData('offset', Tracing._msToSec(measure.duration));
+      }
+      performance.clearMeasures(name);
+    }
+  }
+
+  /**
+   * Converys ms time to s with ms precison
+   * @param time time in ms
+   */
+  private static _msToSec(time: number): number {
+    return time / 1000;
+  }
```

Comment:
I'd imagine we must already have this somewhere else? Though I prefer repeating it here than adding a dependency on a leftpad :P